src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -213,7 +213,7 @@ namespace AirLoopHVACDOAS {
         std::string cCurrentModuleObject = "AirLoopHVAC:Mixer";
         std::string cFieldName;
 
-        auto const instances = state.dataInputProcessing->inputProcessor->epJSON.find(cCurrentModuleObject);
+        auto const &instances = state.dataInputProcessing->inputProcessor->epJSON.find(cCurrentModuleObject);
```

Comment:
`find` returns an iterator, which is a `std::pair` (i.e., two elements). Preferred to do this by value actually. By the way, std::string_view and gsl::span are also iterators so those should be values as well although in those cases you should use std::string_view and gsl::span rather than `auto`.